src/net.cpp

```diff
@@ -1643,6 +1630,10 @@ void CConnman::ThreadDNSAddressSeed()
                     found++;
                 }
                 addrman.Add(vAdd, resolveSource);
+            } else {
+                // We now just ignore DNS Seeds which do not support service
```

Comment:
In commit "Fall back to oneshot for DNS Seeds which don't support filtering."

Perhaps this is obvious with more context, but I'd think "ignoring" is dropping the seed rather than calling AddOneShot. If this said "fall back to one shot" instead of "ignoring" or said how one shot is the same as ignoring, it might be clearer.